pubsub/pulsar/pulsar.go

```diff
@@ -54,6 +55,15 @@ func parsePulsarMetadata(meta pubsub.Metadata) (*pulsarMetadata, error) {
 		}
 		m.EnableTLS = tls
 	}
+	// DisableBatching is defaultly disable batching.
+	m.DisableBatching = true
+	if val, ok := meta.Properties[disableBatching]; ok {
+		enableBatching, err := strconv.ParseBool(val)
```

Comment:
please rename variable to `disableBatch` or something similar to `disableBatching`. The complete opposite of that does create some confusion.